src/components/Tables/ReactiveTableHeader/ReactiveTableHeader.tsx

```diff
@@ -1,6 +1,14 @@
 import React from "react";
+import styled from "@emotion/styled";
 import { useMediaQuery } from "react-responsive";
 import { screenWidths } from "../../../styles/mediaBreakpoints";
+import { fontSizes } from "../../../styles/fonts";
+
+const TableHeader = styled.div({
+  padding: 8,
+  fontSize: fontSizes.font_size_6,
+  fontWeight: 600,
+});
```

Comment:
why not `styled.h6`?

Reply:
h6 would skip the order. the pages goes h1, h2, h6

Reply:
Are we not supposed to skip order?

Reply:
Storybook gets mad if you do. Presumably other things throw warnings as well.

Reply:
Order should not be skipped, for accessibility reasons